doc/source/whatsnew/v0.18.0.txt

```diff
@@ -86,13 +101,78 @@ In addition, ``.round()`` will be available thru the ``.dt`` accessor of ``Serie
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _whatsnew_0180.api_breaking.extract:
+
+Changes to str.extract
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``.str.extract`` method takes a regular expression with capture
+groups, finds the first match in each subject string, and returns the
+contents of the capture groups.
 
+**PRIOR** to 0.18.0, ``extract`` could return a ``Series``, ``Index``,
```

Comment:
use the 'Previous Behavior'/'New Behavior' tags (see 0.17.0.txt for how this is formatted)